src/App.tsx

```diff
@@ -1,39 +1,76 @@
-import React from 'react';
+import { Component } from 'react';
+
 import './App.scss';
 
-function getRandomName(): string {
-  const value = Date.now().toString().slice(-4);
+import { Clock } from './components/Clock';
+
+const names = [
```

Comment:
usually, we move such hardcoded data to `clock. constants.ts` in your case, JFYI